lib/mayaUsd/resources/ae/usdschemabase/ae_template.py

```diff
@@ -208,19 +223,72 @@ def onCreate(self, *args):
             typeNameStr = str(typeName.scalarType)
             typeNameStr += ("[" + str(len(values)) + "]") if hasValue else "[]"
 
-            mdLabel = getPrettyName(self.attrName) if self.useNiceName else self.attrName
-            cmds.textFieldGrp(editable=False, label=mdLabel, text=typeNameStr, annotation=attr.GetDocumentation())
-
-            if hasValue:
-                cmds.popupMenu()
-                cmds.menuItem( label="Copy Attribute Value",   command=lambda *args: setClipboardData(str(values)) )
-                cmds.menuItem( label="Print to Script Editor", command=lambda *args: MGlobal.displayInfo(str(values)) )
+            attrLabel = getPrettyName(self.attrName) if self.useNiceName else self.attrName
+            singleWidgetWidth = mel.eval('global int $gAttributeEditorTemplateSingleWidgetWidth; $gAttributeEditorTemplateSingleWidgetWidth += 0')
+            with AEUITemplate():
+                # See comment in ConnectionsCustomControl below for why nc=5.
+                rl = cmds.rowLayout(nc=5, adj=3)
+                with LayoutManager(rl):
+                    cmds.text(nameTxt, al='right', label=attrLabel, annotation=attr.GetDocumentation())
+                    cmds.textField(attrTypeFld, editable=False, text=typeNameStr, width=singleWidgetWidth*1.5)
```

Comment:
Split the single textFieldGrp control into separate text & textField controls. This way we can set the size of the textField (to be 1.5 times the normal AE single widget width) as requested by design. Also attach the documentation only to the text and the popup only to the field.